### PR TITLE
Security Notes

### PR DESCRIPTION
**Pull request by VinayaSathyanarayana**
Add Security.txt (Production) and Security.MD (Open Source)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1026,6 +1026,18 @@ All statements above will return false if used with `===`
 
 <p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
 
+## ![✔] 6.26. Have a SECURITY.TXT File [PRODUCTION]
```
Comment (kevynb) [inline README.md]: Seeing https://securitytxt.org/ I understand that the filename should be in lowercase.

They also say that the preferred location is under the .well_known directory.

Additionally I think that we should include the link for further reference.

What do you think of that?

Comment (VinayaSathyanarayana) [reply README.md]: Agree with @kevynb . Will update.

Comment (VinayaSathyanarayana) [reply README.md]: Updated the file with references. @kevynb Can be merged

**Comment by goldbergyoni** [comment]
@VinayaSathyanarayana Great topic! Formatting-wise, we don't link from 'Read more' to other sites, only locally or avoid putting reading more. Do you have enough ideas to write about in a dedicated page?

I would also remind the term 'Bug bounty' which is very popular, there must be great quotes on this matter

@js-kyle Assigning you to the PR as it deals with security

And of course, it's always important to get @lirantal take on security matters

**Comment by js-kyle** [comment]
These are great! - I think we should add the bullets to the collection of generic security practices, as this doesn't relate specifically to Node.js, but all types of web app(similar to the PII PR which you have open @VinayaSathyanarayana )

File: README.md [hunk 2]
```diff
@@ -1026,6 +1026,24 @@ All statements above will return false if used with `===`
 
 <p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
 
+## ![✔] 6.26. Have a security.txt File [PRODUCTION]
+
+**TL;DR:** Have a text file called ```security.txt``` under ```/.well-known```  directory (/.well-known/security.txt) or in the root directory (/security.txt) of your website or your Web Application in Production. ```security.txt``` file should contain details using which Security Researchers can report Vulnerabilities and also the contact details of the responsible person/group (email id and/or phone numbers) to whom the reports have to be sent.
```
Comment (lirantal) [inline README.md]: I think we can have the language here as lower case in most cases, for example: `Security Researchers` , `Vulnerabilities`, `Production`

**Comment by lirantal** [comment]
Good stuff.

I've got [express-security-txt](https://www.npmjs.com/package/express-security-txt) that is mostly kept up to date with the security.txt spec (it's not finalized yet) if you wanted to get a look at what data is there to mention here.

**Comment by VinayaSathyanarayana** [comment]
Will update over the weekend

**Comment by goldbergyoni** [comment]
@js-kyle @VinayaSathyanarayana Maybe the inclusion of express/others example makes this a bullet that is not generic rather can be examplified with Node.js?

**Comment by goldbergyoni** [comment]
@js-kyle Can I merge this?

**Comment by VinayaSathyanarayana** [comment]
Please merge

**Comment by js-kyle** [comment]
I still think these are generic tips - they aren't related to Node.js. We could move them to the generic security list as mentioned above

**Comment by VinayaSathyanarayana** [comment]
OK... We can move them to the Generic List

**Comment by goldbergyoni** [comment]
@js-kyle Your call, let's merge

**Comment by VinayaSathyanarayana** [comment]
Let us get this merged

**Comment by js-kyle** [comment]
The feedback on the PR needs to be addressed before we can merge this

File: README.md (hunk 2 above)
Comment (VinayaSathyanarayana) [reply README.md]: Updated

**Comment by VinayaSathyanarayana** [comment]
Corrected the uppercase to lower case. Let us merge

**Comment by js-kyle** [comment]
@VinayaSathyanarayana We are keen to merge this, could we please move these to the generic security practices (see https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/security/commonsecuritybestpractices.md)

The reason for this request, is because the security.txt & security.md items are a general security suggestion as opposed to something that relates to Node.js

**Comment by VinayaSathyanarayana** [comment]
Ok. Will update

**Comment by VinayaSathyanarayana** [comment]
@js-kyle Created a new PR https://github.com/goldbergyoni/nodebestpractices/pull/683 which can be merged. We can close this after merging the other one.

**Comment by js-kyle** [comment]
The new PR has been merged

**Pull request closed without merging**